grpc-sys/build.rs

```diff
@@ -12,14 +12,47 @@
 // limitations under the License.
 
 extern crate gcc;
+extern crate cmake;
+
+use std::path::Path;
+use std::process::Command;
 
 fn main() {
-    // TODO: support static link
+    println!("cargo:rerun-if-changed=build.rs");
+    println!("cargo:rerun-if-changed=grpc/");
+
+    if !Path::new("grpc/.git").exists() {
+        let _ = Command::new("git").args(&["submodule", "update", "--init"])
+                                   .status();
+        let grpc_submodules = [
+            "third_party/zlib",
+            "third_party/protobuf",
+            "third_party/boringssl",
+            "third_party/gflags",
+            "third_party/benchmark"
+        ];
+        for submodule in &grpc_submodules {
+            let _ = Command::new("git").args(&["submodule", "update", "--init", submodule])
```

Comment:
What if L25 succeeds but here fails, will it reach here when try to build again?